src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,44 @@
+package mate.academy.service
+
+import mate.academy.figure.Circle
+import mate.academy.figure.Figure
+import mate.academy.figure.Rectangle
+import mate.academy.figure.Square
+import mate.academy.figure.IsoscelesTrapezoid
+import mate.academy.figure.RightTriangle
+import mate.academy.model.Color
+import kotlin.random.Random
+
+const val START_RANDOM_NUMBER = 1.0
+const val FINISH_RANDOM_NUMBER = 20.0
+const val DEFAULT_RADIUS = 10.0
+const val QUANTITY_FIGURES = 5
+const val ROUND_NUMBER = 10
+const val ROUND_NUMBER2 = 10.0
+const val ISOSCELES_TRIANGLE = 4
+const val RIGHT_TRIANGLE = 3
+const val RECTANGLE = 2
+const val SQUARE = 1
+const val CIRCLE = 0
+
+class FigureSupplier: ColorSupplier() {

```

Comment:
It is better do not inherit ColorSupplier here as this may be cause unexpected behaviour.
If you need ColorSupplier functionality it is ok if it would be just variable